mdformat_pyproject/plugin.py

```diff
@@ -50,11 +53,11 @@ def _parse_pyproject(pyproject_path: pathlib.Path) -> Optional[Mapping]:
     if options is not None:
         mdformat._conf._validate_keys(options, pyproject_path)
         mdformat._conf._validate_values(options, pyproject_path)
-        return options
+    return options
```

Comment:
Super sytle nit: Would you mind adding a blank line above this one?

I like to have a blank line delimiting the end of indented blocks when there is more than one line in the block.

Reply:
```suggestion

    return options
```